src/main/java/io/jenkins/plugins/orka/client/ConfigurationRequest.java

```diff
@@ -26,11 +26,23 @@
     @SuppressFBWarnings("URF_UNREAD_FIELD")
     private int cpuCount;
 
+    @SuppressFBWarnings("URF_UNREAD_FIELD")
+    private String scheduler;
+
     public ConfigurationRequest(String vmName, String image, String baseImage, String configTemplate, int cpuCount) {
+        this(vmName, image, baseImage, configTemplate, cpuCount, null);
+    }
+
+    public ConfigurationRequest(String vmName, String image, String baseImage, String configTemplate, int cpuCount,
+            String scheduler) {
         this.vmName = vmName;
         this.image = image;
         this.baseImage = baseImage;
         this.configTemplate = configTemplate;
         this.cpuCount = cpuCount;
+
+        if (scheduler != "config-default" && scheduler != null) {
```

Comment:
`nit`: I would extract the string to a const

Reply:
In fact I believe we can handle this in a cleaner way.
[Here](https://github.com/jenkinsci/macstadium-orka-plugin/pull/81/files#diff-73b5af94eccd97e52cbb2656aa91a19b0c335ffcc0a1bfbb2069f20a4415f6dbR83).
`ListBoxModel` has an `add` overload that allows you to set a key-value pair of `displayName` and `value`. So for a display name we can use `config-default`, but the value would be `null`. This would eliminate the need for this logic